packages/Modal/src/Modal.stories.tsx

```diff
@@ -249,6 +250,14 @@ export const WithActions = {
               Cancel
             </Button>
           }
+          tertiaryAction={
+            <Hyperlink
```

Comment:
You should put a ghost danger button, it's not a hopper design but it's still better than an hyperlink

Reply:
ok I'll switch it